### PR TITLE
Fix REST API doc typos, punctuation, caps, & wording

### PR DESCRIPTION
**Pull request by chenette**
Edited the RST doc and autogenerated content for the REST API documentation.

Note: This PR is based on #1211 - see that PR to review the changes in ubuntu.rst.

### CODE REVIEW

File: docs/source/app_developers_guide/ubuntu.rst [hunk 1]
```diff
@@ -156,31 +151,29 @@ ends with lines similar to these:
...
-  The validator can process transactions in serial (the default) or parallel
-  with no difference in the state produced. To process in parallel, use the
-  option ``--scheduler parallel``. To get the most benefit from the parallel
-  option, start multiple transaction processors for the types of transactions
-  where there is an expected high volume.
+The validator can process transactions in serial (the default) or parallel
+with no difference in the state produced. To process in parallel, use the
+option ``--scheduler parallel``. To get the most benefit from the parallel
+option, start multiple transaction processors for the types of transactions
+where there is an expected high volume.
```
Comment (aludvik) [inline docs/source/app_developers_guide/ubuntu.rst]: Should this change be in the previous commit?

File: docs/source/app_developers_guide/ubuntu.rst [hunk 2]
```diff
@@ -519,9 +509,8 @@ The output of the command will be similar to this:
...
-To stop the validator, enter CTRL-c in the validator's terminal
-window. A single CTRL-c starts a graceful shutdown.
-Enter multiple CTRL-c characters to force the validator to stop quickly.
+To stop the validator, enter CTRL-c several times in the validator's terminal
+window.
 
-Stop the REST API and transaction processors by by entering
-CTRL-c in the appropriate windows.
+Stop the REST API and transaction processors by entering a single CTRL-c in
+the appropriate windows.
```
Comment (aludvik) [inline docs/source/app_developers_guide/ubuntu.rst]: Should these changes be in the previous commit?

File: docs/source/app_developers_guide/ubuntu.rst (hunk 1 above)
Comment (chenette) [reply docs/source/app_developers_guide/ubuntu.rst]: This isn't a technical change - I just moved some words around (put the default info in the first sentence).  I intended this commit to have the "rewrite" changes.  The other one is the "fix typos" commit.

File: docs/source/rest_api/error_codes.rst [hunk 3]
```diff
@@ -46,12 +46,12 @@ include a single *error* property with three values:
...
    * *title* (string) - a short headline for the error
-   * *message* (string) - a longer human readable description of what went wrong
+   * *message* (string) - a longer, human-readable description of what went wrong
 
 .. note::
 
-   While the title and message may change in the future, **the error code is
-   fixed**, and will always refer to this particular problem.
+   While the title and message may change in the future, the error code
+   will **not** change; it will always refer to this particular problem.
```
Comment (delventhalz) [inline docs/source/rest_api/error_codes.rst]: For what it's worth, the term _"fixed"_ means more to me than _"will not change"_. The first feels like a contractual definition, while the second is a vague aspiration. Perhaps the clause after the semicolon could become: `; it is fixed and will always refer...`

File: docs/source/app_developers_guide/ubuntu.rst (hunk 1 above)
Comment (chenette) [reply docs/source/app_developers_guide/ubuntu.rst]: Following up to say that yes, this change SHOULD have been in the other PR, but wasn't (as you helpfully noted).  Thanks for this catch!  The original ubuntu.rst changes were already merged, and I'm closing this confusing PR, so I'll make this correction in a later PR.  Thanks again!

**Comment by chenette** [comment]
I'm abandoning (closing) this PR because it's too confusing.  I created a new PR with only the REST API changes (the commit to "Fix typos, punctuation, and wording issues").  I've copied Zac's comments to the new PR and will make the fix there - please see #1242 .

**Pull request closed without merging**